api/api/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for api project.
+
+Generated by 'django-admin startproject' using Django 2.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+from config.py import *
```

Comment:
import should be on the end of `settings.py` file else the settings in `config.py` will be overwritten by the ones in the `settings.py`. Also it should check if `config.py` exists then import it.